### PR TITLE
feat(notification): Update copy and examples

### PR DESCRIPTION
**Pull request by adamledwards**

### CODE REVIEW

File: src/site/_includes/moduk/components/notification-banner/success.njk [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{% from "moduk/components/notification-banner/macro.njk" import modukNotificationBanner -%}
```
Comment (jpveooys) [inline src/site/_includes/moduk/components/notification-banner/success.njk:1]: Is this needed – seems identical to the existing success example in moduk-frontend?

Comment (adamledwards) [reply src/site/_includes/moduk/components/notification-banner/success.njk:1]: I didn't see it  let me look again

Comment (adamledwards) [reply src/site/_includes/moduk/components/notification-banner/success.njk:1]: Ah yes ok, must have missed it

**Review by jpveooys — APPROVED** [review]
:+1:

**Pull request merged**